docs/CONTRIBUTING.md

```diff
@@ -143,7 +143,7 @@ See the section on [README](../README.md#getting-started)
 
 ### Configuration
 
-A configmap called `kogito-serverless-operator-builder-config` will be created under the `kogito-workflows` namespace
+A configmap called `sonataflow-operator-builder-config` will be created under the `kogito-workflows` namespace
```

Comment:
According to the `operator.yaml` it is `sonataflow-operator-system`.

```suggestion
A configmap called `sonataflow-operator-builder-config` will be created under the `sonataflow-operator-system` namespace
```

Reply:
This was wrong before! 😓